src/components/backend-ai-settings-view.ts

```diff
@@ -351,6 +373,85 @@ export default class BackendAiSettingsView extends BackendAIPage {
             </div>
           </div>
         </lablup-activity-panel>
+        <backend-ai-dialog id="env-dialog" fixed backdrop persistent closeWithConfirmation>
+          <span slot="title" class="horizontal layout center">${_tr('settings.ConfigPerJobSchduler')}</span>
+          <span slot="action">
+            <mwc-icon-button icon="info" @click="${(e) => this._showConfigDescription(e, 'default')}" style="pointer-events:auto;"></mwc-icon-button>
+          </span>
+          <div slot="content" id="env-container" class="vertical layout centered" style="width: 100%;">
+            <mwc-select
+              id="scheduler-switch"
+              required
+              label="${_t('settings.Scheduler')}" 
+              style="margin-bottom: 10px;"
+              validationMessage="${_t('settings.SchedulerRequired')}"
+              @selected="${(e) => this.changeSelectedScheduleType(e)}">
+              ${this.jobschedulerType.map((item) => html`
+                <mwc-list-item value="${item}">
+                  ${item}
+                </mwc-list-item>`)}
+            </mwc-select>
+            <h4>${_t('settings.SchedulerOptions')}</h4>
+            <div class="horizontal center layout flex row">
+              <span slot="title">${_t('settings.SessionCreationRetries')}</span>
+              <mwc-icon-button icon="info" @click="${(e) => this._showConfigDescription(e, 'retries')}" style="pointer-events:auto;"></mwc-icon-button>
+              <mwc-textfield  id="num-retries"
+                              outlined
+                              required
+                              autoValidate
+                              validationMessage="${_t('settings.InputRequired')}"
+                              type="number"
+                              pattern="[0-9]+"
```

Comment:
@Sujin-Kim1
As you've already seen there's[ a property called `type` in mwc-textfield component](https://github.com/material-components/material-components-web-components/tree/master/packages/textfield).
This also automatically ignores invalid input such as alphabetic character.